examples/Identifying Local Environments in a Complex Crystal.ipynb

```diff
@@ -0,0 +1,445 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig, ax = plt.subplots()\n",
+    "ax.scatter(range(0, env_cluster.num_clusters, 1), num_particles_list)\n",
+    "ax.set_xlabel(\"Cluster keys\")\n",
+    "ax.set_ylabel(\"Number of environment neighbors\")\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "48729e4e",
+   "metadata": {},
+   "source": [
+    "From the figure above, we can see that indeed the particles have different coordination numbers. Next we use `gsd` to save the snapshot and `fresnel` to visualize it."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "32c69064",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import gsd.hoomd\n",
+    "\n",
+    "name = \"gb\"\n",
+    "typeid = env_cluster.cluster_idx.copy()\n",
+    "snapshot = gsd.hoomd.Snapshot()\n",
+    "snapshot.particles.N = np.shape(points)[0]\n",
+    "snapshot.particles.position = points\n",
+    "snapshot.particles.types = [str(i) for i in np.unique(typeid)]\n",
+    "snapshot.particles.typeid = typeid\n",
+    "snapshot.particles.diameter = [1.0] * np.shape(points)[0]\n",
+    "snapshot.configuration.box = [box.Lx, box.Ly, box.Lz, box.xy, box.xz, box.yz]\n",
+    "with gsd.hoomd.open(f\"{name}_lattice.gsd\", mode=\"wb\") as f:\n",
```

Comment:
The `name` is only used once. Delete the variable and define it inline as `gb_lattice.gsd`.